src/mappings/marketplace/events/counter_offer_placed.ts

```diff
@@ -118,6 +118,7 @@ export async function counterOfferPlaced(
                 account: account.id,
                 listing: listing.id,
                 extrinsic: item.extrinsic.id,
+                tokenId: `${listing.makeAssetId.collection.collectionId}-${listing.makeAssetId.tokenId}`,
```

Comment:
for offers it would be takeAssetId
```suggestion
                tokenId: listing.takeAsserId.id,
```